pkg/diff/diff.go

```diff
@@ -316,6 +320,9 @@ func structuredMergeDiff(p *SMDParams) (*DiffResult, error) {
 
 	gvk := p.config.GetObjectKind().GroupVersionKind()
 	pt := gescheme.ResolveParseableType(gvk, p.gvkParser)
+	if pt == nil {
+		return nil, fmt.Errorf("unable to resolve parseableType for gvk: %s", gvk)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to resolve parseableType for gvk (Group Version Kind): %s", gvk)
```
Same

Reply:
@todaywasawesome This is an internal error message but I agree with making it a bit more clear. I added a small modification. 
PTAL